vendor/golang.org/x/crypto/md4/example_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package md4_test
+
+import (
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/md4"
+)
+
+func ExampleNew() {
```

Comment:
__[go_vet]__
* [vet] ExampleNew refers to unknown identifier: New

<!-- Comment by [SideCI](https://sideci.com) -->